code_examples/sdk_examples/src/core_features/messaging/01_generate_message.ts

```diff
@@ -0,0 +1,39 @@
+import * as Kilt from '@kiltprotocol/sdk-js'
+
+export async function generateMessage(
```

Comment:
Since the content of this message is specific, can you change the name of the function to something that reflects this?

Reply:
I have updated it.